archlinuxcn/shadowsocks-rust-git/PKGBUILD

```diff
@@ -0,0 +1,41 @@
+# Maintainer: DuckSoft <realducksoft at gmail dot com>
+# Contributor: rustemb <rustemb at systemli dot org>
+pkgname=shadowsocks-rust-git
+pkgver=1.10.9.r5.g89b329c
+pkgrel=1
+provides=(shadowsocks-rust)
+conflicts=(shadowsocks-rust)
+pkgdesc='Rust port of Shadowsocks'
+arch=('x86_64')
+url='https://github.com/shadowsocks/shadowsocks-rust'
+license=('MIT')
+depends=('openssl')
+makedepends=('cargo' 'libsodium')
+source=("git+$url.git" shadowsocks-rust{,-server}@.service)
+
+pkgver() {
+    cd "${pkgname%-git}"
+    git describe --long --tags | sed 's/^v//;s/\([^-]*-g\)/r\1/;s/-/./g'
+}
+
+build() {
+ 	cd "${pkgname%-git}"
+    cargo build --release
+}
+
+package() {
+	install -Dm644 shadowsocks-rust{,-server}@.service -vt "$pkgdir"/usr/lib/systemd/system/
+
+ 	cd "${pkgname%-git}"
+ 	install -Dm644 LICENSE                             -vt "$pkgdir"/usr/share/licenses/shadowsocks-rust/
+	install -Dm644 examples/config{,_ext}.json         -vt "$pkgdir"/etc/shadowsocks-rust/
+	
+ 	cd target/release
+ 	for name in sslocal ssserver ssurl ssmanager; do
+        install -Dm755 -v $name "$pkgdir"/usr/bin/${name}-rust
+    done
+}
+
+b2sums=('SKIP'
```

Comment:
what is it? `b2sums`?

Reply:
https://en.wikipedia.org/wiki/BLAKE_(hash_function)

Reply:
Well then, learn something new. :)